catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/table/table-viz.view.js

```diff
@@ -80,5 +107,48 @@ module.exports = Marionette.LayoutView.extend({
         }), {
             replaceElement: true
         });
+    },
+    exportAll: function() {
+        let data = {
+            hiddenFields: [],
+            columnOrder: user.get('user').get('preferences').get('columnOrder'),
+            columnAliasMap: properties.attributeAliases,
+            metacards: store.get('content').getCurrentQuery().get('result').get('results').fullCollection
+        };
+        $.ajax({
+            type: "POST",
+            url: '/search/catalog/internal/transform/csv?_=' + Date.now(),
```

Comment:
It was not being added automatically. I had to add this line before it showed up.